File: rest/taskrouter/workers/list/post/example-1/example-1.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require_once('/path/to/twilio-php/Services/Twilio.php'); // Loads the library

// Your Account Sid and Auth Token from twilio.com/user/account
$accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$authToken = "{{ auth_token }}";
$workspaceSid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";

$client = new TaskRouter_Services_Twilio($accountSid, $authToken, null);

$params = array();
$params['Attributes'] = '{"type":"support"}';

$worker = $client->workspace->workers->create("Support Worker 1", $params);
echo $worker->friendly_name;

File: rest/taskrouter/workflows/instance/get/example-1/example-1.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'rubygems'          # This line not needed for ruby > 1.8
require 'twilio-ruby'

# Get your Account Sid and Auth Token from twilio.com/user/account
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = '{{ auth_token }}'
workspace_sid = 'WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
workflow_sid = '{{ workflow_sid }}'

client = Twilio::REST::TaskRouterClient.new account_sid, auth_token, workspace_sid

workflow = client.workspace.workflows.get(workflow_sid)
puts workflow_sid.friendly_name
